### PR TITLE
Fix Kotlin bug at handling if statement blocks containing comments and statements

### PR DESCRIPTION
**Pull request by ketkarameya**
We were not handling scenarios when blocks contained statements and comments.
The problem is that in tree-sitter-kotlin `comment` is not subtype of `statement`. Its a sibling type. (if it makes sense).
Therefore I had to make it `(_)*` because `(_)` would only match first statement(s) or comment(s).

----
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #419

### CODE REVIEW

File: src/cleanup_rules/kt/rules.toml [hunk 1]
```diff
@@ -66,7 +66,7 @@ query = """
 (
 (if_expression (_) (_) 
                (control_structure_body ((if_expression [((boolean_literal) @condition) (parenthesized_expression (boolean_literal) @condition)] 
-               (control_structure_body (_) ) @consquent
```
Comment (ketkarameya) [inline src/cleanup_rules/kt/rules.toml:69]: remove unnecessary match structure. (could be a different PR but ...)

File: src/cleanup_rules/kt/rules.toml [hunk 2]
```diff
@@ -86,7 +86,7 @@ query = """
 (
 (if_expression (_) (_) 
                (control_structure_body ((if_expression [((boolean_literal) @condition) (parenthesized_expression (boolean_literal) @condition)] 
-               (control_structure_body (_) ) @consquent)) ) @alternative) @if_expression
```
Comment (ketkarameya) [inline src/cleanup_rules/kt/rules.toml:89]: remove unnecessary match structure. (could be a different PR but ...)

File: src/cleanup_rules/kt/rules.toml [hunk 3]
```diff
@@ -125,8 +125,8 @@ name = "simplify_if_true_with_alternative"
 query = """
 (
 (if_expression [((boolean_literal) @condition) (parenthesized_expression (boolean_literal) @condition)] 
-               (control_structure_body (_) @consequence ) 
-               (control_structure_body (_) ) @alternative) @if_expression
+               (control_structure_body (_)* @consequence )
```
Comment (ketkarameya) [inline src/cleanup_rules/kt/rules.toml:128]: bug fix

File: src/cleanup_rules/kt/rules.toml [hunk 4]
```diff
@@ -149,8 +149,8 @@ name = "simplify_if_false_with_alternative"
 query = """
 (
 (if_expression [((boolean_literal) @condition) (parenthesized_expression (boolean_literal) @condition)] 
-               (control_structure_body (_)? ) @consquent
-               (control_structure_body (_)? @alternative) ) @if_expression
+               (control_structure_body) @consquent
+               (control_structure_body (_)* @alternative) ) @if_expression
```
Comment (ketkarameya) [inline src/cleanup_rules/kt/rules.toml:153]: bug fix

**Review by ketkarameya — COMMENTED** [review]
Annotated the PR

**Review by lazaroclapp — APPROVED** [review]
I think I get it and the tests look good, so 👍
